src/services/Friend/HandleGroup.js

```diff
@@ -181,6 +259,14 @@ export async function leaveGroup(groupID, app = firebase) {
     .then(() => {
       Alert.alert("You have successfully left this group.");
     })
+    .then(() => {
+      sendNotificationToAllGroupMembers(
+        groupID,
+        userID,
+        "Chattoku's Group Member Left",
+        `${username} has left ${groupName} group.`
+      );
```

Comment:
I don't think this is necessary either. It might be better to show it in the chat history instead since most messenger app do it this way, so the users might feel more used to it (no need to do it right now, can do it in the future)